interbotix_ros_xseries/interbotix_xs_sdk/src/xs_sdk_obj.cpp

```diff
@@ -1086,6 +1088,22 @@ bool InterbotixRobotXS::robot_srv_get_motor_registers(interbotix_xs_msgs::Regist
   return true;
 }
 
+/**
+ * @brief ROS Service to set the gripper calibration value in calibration map
+ *
+ * @param req
+ * @param res
+ * @return true
+ * @return false
+ * @details - refer to the service definition for details
+ */
```

Comment:
Add function docs, match style of other function docs

Reply:
Added